WordPress/gutenberg
# [RNMobile] Correct isMobile condition in nested Media&Text

## Description

During the work over `Column` mobile block turned out that nested `Media&Text` component is not presented in a stacked layout. The issue was caused by the wrong assumption where we were based on property `isMobile` from `withViewportMatch` which was calculated on device width, however, in our case, we should rely on a parent

## How has this been tested?

**NOTE** I've merged these changes into `rnmobile/column-block` branch to test it properly.

To test:

1. Add column block
2. Choose 2 columns
3. Add Media&Text
4. Observe if Media&Text is in a stacked layout

## Screenshots 

before | after
--- | ---
![unknown](https://user-images.githubusercontent.com/22746080/72805071-4ae92680-3c52-11ea-8e85-a4eb1871cec3.png) | ![unknown (1)](https://user-images.githubusercontent.com/22746080/72805906-3c037380-3c54-11ea-88d2-9d227329175e.png)


## Types of changes





## Checklist:
